src/getDefaultTagStructureForMode.js

```diff
@@ -461,6 +459,50 @@ const getDefaultTagStructureForMode = (mode) => {
       ]),
     ],
 
+    [
+      'link', new Map([
+        // Signature seems to require a namepath OR URL and might be checked as such.
+        [
+          'nameContents', 'namepath-referencing',
+        ],
+
+        // "namepath"
+        [
+          'typeOrNameRequired', true,
+        ],
```

Comment:
I know my comment stated `typeOrNameRequired `, but if we're treating the namepath/URL as a name, wouldn't this make more sense as `nameRequired`?

Reply:
Probably, yeah.